inmem/dbrp_mapping_test.go

```diff
@@ -18,26 +18,31 @@ func initDBRPMappingService(f platformtesting.DBRPMappingFields, t *testing.T) (
 }
 
 func TestDBRPMappingService_CreateDBRPMapping(t *testing.T) {
+	t.Skip("not needed anymore because we are moving to a new implementation of dbrp")
```

Comment:
@gianarb it would be best to have this set of tests with the new dbrp mapping service 👍

Reply:
Tests like this Test_handlePostDBRP in `dbrp` convers the service as well. I do not see a need for that